[2023-09-10T13:16:52.997+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: transactions.insert_into_transactions scheduled__2023-09-09T00:00:58+00:00 [queued]>
[2023-09-10T13:16:53.004+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: transactions.insert_into_transactions scheduled__2023-09-09T00:00:58+00:00 [queued]>
[2023-09-10T13:16:53.005+0000] {taskinstance.py:1361} INFO - Starting attempt 2 of 2
[2023-09-10T13:16:53.018+0000] {taskinstance.py:1382} INFO - Executing <Task(GenericTransfer): insert_into_transactions> on 2023-09-09 00:00:58+00:00
[2023-09-10T13:16:53.026+0000] {standard_task_runner.py:57} INFO - Started process 4077 to run task
[2023-09-10T13:16:53.030+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'transactions', 'insert_into_transactions', 'scheduled__2023-09-09T00:00:58+00:00', '--job-id', '105', '--raw', '--subdir', 'DAGS_FOLDER/transactions.py', '--cfg-path', '/tmp/tmp_ua13xkn']
[2023-09-10T13:16:53.035+0000] {standard_task_runner.py:85} INFO - Job 105: Subtask insert_into_transactions
[2023-09-10T13:16:53.087+0000] {task_command.py:415} INFO - Running <TaskInstance: transactions.insert_into_transactions scheduled__2023-09-09T00:00:58+00:00 [running]> on host 80a7ead3f1b6
[2023-09-10T13:16:53.154+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Aditiya Rahman' AIRFLOW_CTX_DAG_ID='transactions' AIRFLOW_CTX_TASK_ID='insert_into_transactions' AIRFLOW_CTX_EXECUTION_DATE='2023-09-09T00:00:58+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-09-09T00:00:58+00:00'
[2023-09-10T13:16:53.161+0000] {base.py:73} INFO - Using connection ID 'postgres' for task execution.
[2023-09-10T13:16:53.171+0000] {base.py:73} INFO - Using connection ID 'datawarehouse' for task execution.
[2023-09-10T13:16:53.172+0000] {generic_transfer.py:77} INFO - Extracting data from postgres
[2023-09-10T13:16:53.173+0000] {generic_transfer.py:78} INFO - Executing: 
 
SELECT a."TrackId",
       b."InvoiceId",
       d."CustomerId",
       c."InvoiceDate",
       c."Total",
       concat(d."FirstName", ' ', d."LastName"),
       d."Address",
       d."City",
       d."State",
       d."Country",
       d."PostalCode",
       d."Email"
FROM "Track" a 
JOIN "InvoiceLine" b
ON  a."TrackId" = b."TrackId"
JOIN "Invoice" c
ON  b."InvoiceId" = c."InvoiceId"
JOIN "Customer" d
ON c."CustomerId" = d."CustomerId"
[2023-09-10T13:16:53.176+0000] {base.py:73} INFO - Using connection ID 'postgres' for task execution.
[2023-09-10T13:16:53.181+0000] {sql.py:418} INFO - Running statement: 
SELECT a."TrackId",
       b."InvoiceId",
       d."CustomerId",
       c."InvoiceDate",
       c."Total",
       concat(d."FirstName", ' ', d."LastName"),
       d."Address",
       d."City",
       d."State",
       d."Country",
       d."PostalCode",
       d."Email"
FROM "Track" a 
JOIN "InvoiceLine" b
ON  a."TrackId" = b."TrackId"
JOIN "Invoice" c
ON  b."InvoiceId" = c."InvoiceId"
JOIN "Customer" d
ON c."CustomerId" = d."CustomerId", parameters: None
[2023-09-10T13:16:53.190+0000] {sql.py:427} INFO - Rows affected: 2240
[2023-09-10T13:16:53.199+0000] {generic_transfer.py:95} INFO - Running preoperator
[2023-09-10T13:16:53.199+0000] {generic_transfer.py:96} INFO - ['DROP TABLE IF EXISTS transactions', '\n        CREATE TABLE transactions (\n        Track_Id INT NOT NULL,\n        Invoice_Id INT NOT NULL,\n        Customer_Id INT NOT NULL,\n        Invoice_Date TIMESTAMP NOT NULL,\n        Total NUMERIC(10,2) NOT NULL,\n        Full_Name VARCHAR(50) NOT NULL,\n        "Address" VARCHAR(70),\n        "City" VARCHAR(40),\n        "State" VARCHAR(40),\n        "Country" VARCHAR(40),\n        "PostalCode" VARCHAR(10),\n        "Email" VARCHAR(60) NOT NULL\n        )\n        ']
[2023-09-10T13:16:53.205+0000] {base.py:73} INFO - Using connection ID 'datawarehouse' for task execution.
[2023-09-10T13:16:53.208+0000] {sql.py:418} INFO - Running statement: DROP TABLE IF EXISTS transactions, parameters: None
[2023-09-10T13:16:53.331+0000] {sql.py:418} INFO - Running statement: 
        CREATE TABLE transactions (
        Track_Id INT NOT NULL,
        Invoice_Id INT NOT NULL,
        Customer_Id INT NOT NULL,
        Invoice_Date TIMESTAMP NOT NULL,
        Total NUMERIC(10,2) NOT NULL,
        Full_Name VARCHAR(50) NOT NULL,
        "Address" VARCHAR(70),
        "City" VARCHAR(40),
        "State" VARCHAR(40),
        "Country" VARCHAR(40),
        "PostalCode" VARCHAR(10),
        "Email" VARCHAR(60) NOT NULL
        )
        , parameters: None
[2023-09-10T13:16:53.335+0000] {generic_transfer.py:105} INFO - Inserting rows into datawarehouse
[2023-09-10T13:16:53.342+0000] {base.py:73} INFO - Using connection ID 'datawarehouse' for task execution.
[2023-09-10T13:16:53.487+0000] {sql.py:512} INFO - Loaded 1000 rows into transactions so far
[2023-09-10T13:16:53.635+0000] {sql.py:512} INFO - Loaded 2000 rows into transactions so far
[2023-09-10T13:16:53.669+0000] {sql.py:515} INFO - Done loading. Loaded a total of 2240 rows into transactions
[2023-09-10T13:16:53.676+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=transactions, task_id=insert_into_transactions, execution_date=20230909T000058, start_date=20230910T131652, end_date=20230910T131653
[2023-09-10T13:16:53.726+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-10T13:16:53.737+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
